src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        if (user.getEmail().equals(email) && user.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
email is basically already checked, since we found user by it, so we can remove this check. Also, we can move all checks to a single condition and return the result of it right away without any `if` statements :wink: